compiler/rustc_resolve/src/def_collector.rs

```diff
@@ -423,6 +423,7 @@ impl<'a, 'b, 'tcx> visit::Visitor<'a> for DefCollector<'a, 'b, 'tcx> {
     }
 
     fn visit_crate(&mut self, krate: &'a Crate) {
+        self.resolver.current_crate_outer_attr_insert_span = Some(krate.spans.inject_use_span);
```

Comment:
`krate.spans` should be passed to `Resolver::new` instead.

Reply:
Changed